util/netcdf_generator.py

```diff
@@ -86,9 +86,13 @@ def _filter_params(self, ds, params):
             params_to_filter.append('annotations')
 
         for key in ds.data_vars:
-            if key not in params_to_filter:
+            # do not remove qc parameters '%s_qc_executed' and '%s_qc_results'
+            if key not in params_to_filter and not self._is_qc_parameter(key):
                 ds = ds.drop(key)
         return ds
+        
+    def _is_qc_parameter(self, param):
+        return 'qc_executed' in param or 'qc_results' in param
 
```

Comment:
It appears the 'param" argument is a single value; if that's so it seems this line might read better as
return param in ('qc_executed', 'qc_results')

Reply:
the return statement is performing a string inclusion, not a list inclusion, so it's correct as is

Reply:
The 'param' argument is a single value, but what I'm doing is checking if 'qc_executed' is a part of the param string or 'qc_results' is a part of the param string. E.g. param could be 'pco2_co2flux_qc_results' and I would want a match.